sound/soc/sof/trace.c

```diff
@@ -539,6 +539,10 @@ EXPORT_SYMBOL(snd_sof_trace_notify_for_error);
 
 void snd_sof_release_trace(struct snd_sof_dev *sdev)
 {
+	struct sof_ipc_fw_ready *ready = &sdev->fw_ready;
+	struct sof_ipc_fw_version *v = &ready->version;
+	struct sof_ipc_dma_trace_params_ext params;
```

Comment:
It should be enough to send simple message, no?
`struct sof_ipc_cmd_hdr msg;`

Reply:
a simple message would need to be parsed separately. if it is a trace ipc message, the glb type would already be handled in the FW